halyard-cli/src/main/java/com/netflix/spinnaker/halyard/cli/command/v1/config/providers/AbstractEditFeaturesProviderCommand.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright 2017 Google, Inc.
```

Comment:
This should reference you or your company (and year should be updated).

Reply:
@ezimanyi Thank you for the feedback. Changes done :)